test/box/gh-6247-tx_man-broken-error-message.result

```diff
@@ -0,0 +1,59 @@
+-- test-run result file version 2
+env = require('test_run')
+ | ---
+ | ...
+test_run = env.new()
+ | ---
+ | ...
+test_run:cmd("create server tx_man with script='box/tx_man.lua'")
+ | ---
+ | - true
+ | ...
+test_run:cmd("start server tx_man")
+ | ---
+ | - true
+ | ...
+test_run:cmd("switch tx_man")
+ | ---
+ | - true
+ | ...
+
+s = box.schema.create_space('test')
+ | ---
+ | ...
+pk = s:create_index('pk', {parts={1, 'uint'}})
+ | ---
+ | ...
+sk = s:create_index('sk', {parts={2, 'uint'}})
+ | ---
+ | ...
+
+for i = 1,500 do s:replace{i, i, i} end
```

Comment:
Why do you need so many tuples? Why don't you inline this test into box/tx_man.test.lua?

Reply:
> Why do you need so many tuples?

To make sure that GC will make the tuple {1, 1, 1} clean.

> Why don't you inline this test into box/tx_man.test.lua?

Just tired to rebase tx_man.test.lua. But I ok, I'll move.